_sass/partials/_print-pdf-view.scss

```diff
@@ -13,7 +14,7 @@ $print-pdf-view: true !default;
         prince-pdf-profile: $pdf-profile;
         prince-pdf-display-doc-title: true; // only supported from Prince 13
         // Note colour profiles are stored in the non-generated Jekyll root, not in _site
-        prince-pdf-output-intent: url("../../../_tools/profiles/#{$color-profile}");
+        prince-pdf-output-intent: url("#{$path-to-root-directory}/_tools/profiles/#{$color-profile}");
```

Comment:
@arthurattwell You've gone from three ..'s to two here.

Reply:
@LouiseSteward Nice catch, thanks! Fixed in 49d83ba.